stl/inc/xhash

```diff
@@ -1336,6 +1359,19 @@ public:
         return node_type::_Make(_Ptr, _List._Getal());
     }
 
+#if _HAS_CXX23
+    template <class _Kx, class _Mytraits = _Traits, class = typename _Mytraits::_Transparent,
+        enable_if_t<!disjunction_v<is_convertible<_Kx, const_iterator>, is_convertible<_Kx, iterator>>, int> = 0>
+    node_type extract(_Kx&& _Keyval) {

```

Comment:
if the body here were in a seperate ugly function it could be merged with the above extract(const key_type& _Keyval) method. I don't feel that strongly about this, but it could prevent us from forgetting to change one or the other later.

Reply:
I suppose they already share via _Extract :shrug: